.github/workflows/periodic-snapshot.yml

```diff
@@ -3,7 +3,7 @@ name: Periodic Snapshot
 # 10 minutes after midnight on the first of every month
 on:
   schedule:
-  - cron: "10 0 1 * *"
+    - cron: '10 0 1 * *'
```

Comment:
Just out of curiosity: Was this an accidental change by Linter? Or was it intentional?

Reply:
intentional:

> - some autoformatting was done

Reply:
Oh, I missed that, thanks. :)